docs/examples/example_lime.ipynb

```diff
@@ -244,7 +248,11 @@
     "from metalearners.utils import simplify_output\n",
     "\n",
     "def predict(X):\n",
-    "    return simplify_output(rlearner.predict(X, is_oos=True))"
+    "    X_pd = pd.DataFrame(X, copy=True)\n",
+    "    for c in X_pd.columns:\n",
+    "        # This line sets the cat.categories correctly (even if not all are present in X)\n",
+    "        X_pd[c] = X_pd[c].astype(df[feature_columns].iloc[:, c].dtype)\n",
```

Comment:
```suggestion
    "        X_pd[c] = X_pd[c].astype(df[c].dtype)\n",
```
Would this work, too?

Reply:
No, the issue here is that the numpy array does not have column names so when converting it to a `dataframe` the columns names are `0,1,2...` so we need to use `iloc` to access the columns in the original `dataframe`.

Reply:
That makes perfect sense - thanks for explaining!